src/active_zoneset_node.cpp

```diff
@@ -77,6 +77,11 @@ bool ActiveZonesetNode::isAllInformationAvailable() const
   return active_zoneset_id_.is_initialized() && zoneset_config_.is_initialized();
 }
 
+ZoneSet ActiveZonesetNode::activeZoneset() const
```

Comment:
I think this should be called by the deprecated function.

Reply:
Holds for all future changes. This should also resolve your coverage issues.

Reply:
> This should also resolve your coverage issues.
No it doesn't solve the coverage issues, but I will do it anyway